example.js

```diff
@@ -1,12 +1,12 @@
 'use strict'
```

Comment:
Could you please move this example to the examples folder? I just realised now that it ended up in the root path